.githooks/pre-commit

```diff
@@ -1,9 +1,31 @@
 #!/bin/sh
 echo "Formatting Python code with black"
 black .
-if xmllint --version 2>/dev/null; then
-  echo "Linting XML files"
-  for f in $(find . -name *.xml); do xmllint --format $f > /tmp/$$lint; cp /tmp/$$lint $f; done && rm /tmp/$$lint
+
```

Comment:
Please change above so that your hook runs `black --check .` on the same files you are running your xml linter on, and rejecting the commit if the check fails (meaning that one or more files would be reformatted).